### DISCUSSION TITLE
更新版本以后从平板登录变成手机登录,求解决

### DISCUSSION THREAD

**Discussion by nenemiku** [mamoe/mirai#794] [Q&A] [answered]
请问代码在哪里设置回平板登录,这样的话手机切换账号能不受到影响.谢谢!

**Comment by nenemiku** [comment]
2020-12-30 16:15:11 I/Mirai: Mirai 正在使用桌面环境. 如遇到验证码将会弹出对话框.
 可添加 JVM 属性 `mirai.no-desktop` 以关闭.

**Comment by Him188** [comment, marked as answer]
看新文档 Bots 章节 https://github.com/mamoe/mirai/blob/dev/docs/Bots.md#%E5%88%87%E6%8D%A2%E7%99%BB%E5%BD%95%E5%8D%8F%E8%AE%AE
